src/apple_tv/flow.py

```
@@ -0,0 +1,263 @@
+        self._register_skip_backward()
+        self._register_set_repeat()
+        self._register_set_shuffle()
+        self._register_switch_account()
+
+    async def trigger_companion_link_failed(self, device: AppleTVDevice) -> None:
+        try:
+            card = self._app.homey.flow.get_device_trigger_card(
+                'appletv_companion_link_failed'
+            )
+            await card.trigger(device)
+        except Exception as err:
```

Ruff (0.15.5)

[warning] Do not catch blind exception: `Exception`

(BLE001)